### ISSUE TITLE
Internal prometheus URL doesn't support routePrefix

### ISSUE THREAD

**Issue by usulkies** [opencost/opencost-helm-chart#71]
The kube-prometheus-stack helm chart supports the routePrefix for prometheusSpec.
This means that prometheus endpoint can have a route prefix.
Current code does not support this, but assuming that the internal endpoint always listens on the root path.

https://github.com/opencost/opencost-helm-chart/blob/6ed2eb42ab99d2bc2f699e949a72226a56c6068e/charts/opencost/templates/_helpers.tpl#L78

**Comment by toscott** [comment]
I believe you can leverage the "external" prometheus option to set a more customized url:

```
opencost:
  prometheus:
    external:
      enabled: true
      url: 'http://prometheus.monitoring.svc:9090/custom-path'
```

**Comment by mattray** [comment]
@usulkies do you want to provide a PR or should we close the issue since there's a work-around?

**Comment by usulkies** [comment]
> @usulkies do you want to provide a PR or should we close the issue since there's a work-around?

It's a good question.
There is a simple workaround which I used indeed. But before using it I had to make sure that there are no other consequences for using "external" URL. Only after I verified that it doesn't really matter using internal/external URL - then I felt sure enough working around.
But as for now I think that it doesn't worth the time. So I'll close it, thank you.
